linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/HttpConfigTest.scala

```diff
@@ -46,6 +49,76 @@ class HttpConfigTest extends FunSuite with Awaits {
 
   }
 
+  test("DefaultsTest - no streaming, no streamAfterContentLengthKB") {
+    val yaml = s"""
+                  |protocol: http
+                  |identifier:
+                  |  kind: io.l5d.methodAndHost
+                  |maxInitialLineKB: 4
+                  |servers:
+                  |- port: 5000
+      """.stripMargin
+    val config = parse(yaml)
+
+    val streaming = config.routerParams(Stack.Params.empty)[Streaming]
+    assert(streaming.enabled)
```

Comment:
@zaharidichev Any ideas on how to test the payload here?

Reply:
Unfortunatelly finagle has that private, so I dont see a way of accessing it. Quite a pity indeed.